source/docs/appendices/migration-guides.md

```diff
@@ -19,6 +19,26 @@ While it was possible to fetch configurations from Magento2 by using the GraphQL
 
 Please refer to [Using Magento Configuration](/docs/magento2/using-magento-configuration.html) for further information.
 
+### Magento Admin Detection
+
+It is now possible to detect if the user connected on the shop is an admin that is connected in Magento. Only thing is you must setup the following environment variable: `FRONT_COMMERCE_MAGENTO_ADMIN_TOKEN`.
+
+Please refer to [Detect admin users](/docs/magento2/detect-admin-users.html) for more details.
+
+### Translation locales variants
+
+<blockquote>
+**Important:** Please remove your `.front-commerce/translations` folder after updating to 2.1.0. Otherwise your local build will fail.
+</blockquote>
+
+Previously, the translation mechanism only accepted a main language (`fr`, `en`, ...). It is now possible to set translation files using locales that includes the country (`fr-FR`, `en-US`, `en-GB`, ...).
+
+In new projects, new locales in the `config/stores.js` will automatically create `translations/<new-locale>.json` files so you don't have to worry about it.
+
+In existing projects that already have main language translation files (e.g. `translations/en.json`) new files won't be created automatically. This means that if you have a store using `en-US` and a store using `en-GB` while only having a `translations/en.json` file in your project, both stores will use `translations/en.json`. If you want to add specificities for each locale, please create manually the files with the correct filename (`translations/en-GB.json`, `translations/en-US.json`).
+
+Please note that if some translations are shared across `en-GB` and `en-US`, you can put them in a `translations/en.json`. It will still be used even if you have created `translations/en-GB.json` or `translations/en-US.json`.
```

Comment:
This paragraph would be great to be duplicated on the documentation page as it helps in maintaining translations in such a context.

Reply:
That's what I meant when talking about the short-locale in the translations page.